app/src/main/java/org/fossasia/openevent/activities/SpeakersActivity.java

```diff
@@ -29,7 +30,7 @@
 /**
  * Created by MananWason on 30-06-2015.
  */
-public class SpeakersActivity extends AppCompatActivity implements SearchView.OnQueryTextListener {
+public class SpeakersActivity extends BaseActivity implements SearchView.OnQueryTextListener {
```

Comment:
in BaseActivity we are logging some output based on Activity life-cycle.probably we can have log for speakersActivity also.